### DISCUSSION TITLE
How Microsoft control display breadcrumb max display items?

### DISCUSSION THREAD

**Discussion by giabaoye** [microsoft/fluentui#31335] [Q&A] [answered]
Hi everyone, when I search breadcrumb component in [https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb](url). I found that when I resize the screen smaller, it's auto collapse item like this to fit screen
<img width="1427" alt="Screenshot 2024-05-11 at 12 15 38 AM" src="https://github.com/microsoft/fluentui/assets/21209285/7ad1eb2c-4783-47a7-b2bf-365752f721a4">
<img width="1440" alt="Screenshot 2024-05-11 at 12 15 47 AM" src="https://github.com/microsoft/fluentui/assets/21209285/96c7b7d2-86f5-4d74-a6b9-dce2eda7fe7c">
 
and auto display more item when it has enough space. 
I have a question is how can Microsoft handle this one? Did they have any built-in function for that 
Any help will be appreciated 🙏🏻

**Comment by khmakoto** [comment, marked as answer]
v8's `Breadcrumb` uses [ResizeGroup](https://developer.microsoft.com/en-us/fluentui#/controls/web/resizegroup) under the hood to handle this.

In v9, we also have [Overflow](https://react.fluentui.dev/?path=/docs/components-overflow--default) for this type of scenarios.

**Reply by giabaoye** [reply]
so in v8's `Breadcrumb`, can I customize `ResizeGroup` inside the `Breadcrumb`?

**Reply by khmakoto** [reply]
You can via the `onGrowData` and `onReduceData` props.
